qcodes/instrument_drivers/cryogenic/CryogenicSMS120C.py

```diff
@@ -184,17 +190,39 @@ def _get_persistentMode(self):
         _, value = self.query('HEATER')
         field = self._get_field()
         # check for switch heater OFF, and non-zero current
-        if 'OFF' in value and (field != 0):
+        if 'OFF' in value and abs(field <= 0.007):
+            persistentField = self._get_persistentField()
             units = self._get_unit()
             if units == 1:
-                print("Magnet in persistent mode, at a field of %f T" % field)
+                print("Magnet in persistent mode, at a field of %f T" %
```

Comment:
This print call (and many more below in the driver) should be converted to logging calls with the appropriate level.